webrtc.html

```diff
@@ -1300,14 +1302,14 @@
                                 produce an identity assertion, reject
                                 <var>p</var> with a <code>DOMException</code>
                                 object whose <code>name</code> attribute has
-                                the value <code>NotReadableError</code> and
+                                the value <code>NotReadableError</code>, and
                                 abort these steps.</p>
                               </li>
                               <li>
                                 <p>Reject <var>p</var> with a
                                 <code>DOMException</code> object whose
                                 <code>name</code> attribute has the value
-                                <code>OperationError</code> and abort these
+                                <code>OperationError</code> and, abort these
```

Comment:
Comma in wrong place, though I just noticed this isn't an if-statement, and there are no more steps, so can we simply remove ", and abort these steps?"